strongbox-web-core/pom.xml

```diff
@@ -41,16 +41,10 @@
 
     <properties>
 
-        <!-- Default ports for local development: -->
-        <port.jetty.listen>48080</port.jetty.listen>
-        <port.jetty.shutdown>19081</port.jetty.shutdown>
-
-        <strongbox.host>localhost</strongbox.host>
-        <strongbox.port>${port.jetty.listen}</strongbox.port>
+        <strongbox.port>48080</strongbox.port>
```

Comment:
@yupitsleen : What I meant that this property could be removed from everywhere. :)